packages/admin/admin-theme/src/typographyOptions.ts

```diff
@@ -9,6 +9,7 @@ export const fontWeights = {
     fontWeightRegular: 300,
     fontWeightMedium: 400,
     fontWeightBold: 500,
+    fontWeightVeryBold: 600,
```

Comment:
Should we align the fontWeights with a common name mapping? https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#common_weight_name_mapping

Reply:
Generally I think we should, the problem is, every project would have to change every font-weight used from the theme. Maybe it's possible with a codemod, if not, that would be a lot of work to restore the previous style.

Reply:
Done a129b63d0fb49b3b3e4e5a5d6d0a1527583fc2d1 

@jamesricky I don't think it's used regularly. But we can make a codemod